nucliadb_telemetry/src/sync_telemetry.rs

```diff
@@ -29,11 +48,14 @@ impl SyncTelemetryLoop {
 
 lazy_static::lazy_static! {
     static ref SYNC_TELEMETRY: Option<Mutex<Sender<TelemetryEvent>>> = {
-        if !is_telemetry_enabled() { return None; }
-        let (stxn, rtxn) = channel();
-        let Some(telemtry_loop) = SyncTelemetryLoop::new(rtxn) else { return None };
-        std::thread::spawn(move || telemtry_loop.run());
-        Some(Mutex::new(stxn))
+        if is_telemetry_enabled() {
+            let (stxn, rtxn) = channel();
```

Comment:
Using the conventional `tx` and `rx` variable names would be more understandable here

Reply:
This comment is not treated in your last commit